.github/workflows/build.yml

```diff
@@ -103,55 +104,58 @@ jobs:
       - name: Setup Node/NPM with pinned Volta version
         uses: volta-cli/action@v4.0.0
 
+      - name: Setup Cargo Cache
+        uses: Swatinem/rust-cache@v2
+
       - name: Install Rust
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ env.RUST_VERSION }}
-          components: rustfmt,clippy
-          override: true
-
-      - name: Setup Cargo Cache
-        uses: Swatinem/rust-cache@v2
+          targets: |
+            x86_64-unknown-linux-gnu
+            wasm32-wasi
+          components: rustfmt,clippy,rust-std
 
       - name: Install Dependencies
         uses: bahmutov/npm-install@v1
 
-      - name: Lint Nx
+      - name: Lint Nx Configuration
         run: npx nx workspace-lint
 
-      - name: Build, Test, and Package
-        uses: MansaGroup/nrwl-nx-action@v2
-        with:
-          targets: "build,test,package"
-          all: true
-          # If pushing, exclude hangar because we're going to run it in a separate job
-          args: "${{ github.event_name == 'push' && '--exclude=hangar ' || '' }}--configuration=release --output-style=stream --verbose"
+      - name: Build
```

Comment:
Can we make each one of these an npm script at the root `package.json` and just call that script here?

It's useful to be able to run those on a dev environment

Reply:
There already is a script that does all this (`npm run build`, added by you I believe), but I wouldn't put it in the workflow. The separation is useful because CI excludes hangar here and it uses "release" with other configurations that are appropriate for CI.

Reply:
I meant to add a script for each of these steps separately